POS/Integration/01_ConfigureWarehouse.md

```diff
@@ -2,7 +2,6 @@
 [!!Manage the POS warehouse](./08_ManageWarehouse.md)
 [!!Handle an out-of-stock order](../Troubleshooting/01_OutOfStockOrder.md)
 
```

Comment:
[!!Configure the warehouse groups](../../RetailSuiteWarehousing/Integration/01_ConfigureWarehouseGroups.md)
[!!Configure the warehouses](../../RetailSuiteWarehousing/Integration/02_ConfigureWarehouses.md)